warehouse/query-core/src/main/java/datawave/query/jexl/visitors/PushdownLargeFieldedListsVisitor.java

```diff
@@ -108,13 +149,18 @@ public Object visit(ASTOrNode node, Object data) {
         fields.addAll(rangeNodesByField.keySet());
         
         for (String field : fields) {
+            boolean skip = false;
+            if (this.fields != null && !this.fields.contains(field)) {
+                skip = true;
+            }
             
             Collection<JexlNode> eqNodes = eqNodesByField.get(field);
             Collection<JexlNode> rangeNodes = rangeNodesByField.get(field);
             
             // if "_ANYFIELD_" or "_NOFIELD_", then simply add the subset back into the children list
             // if past our threshold, then add a ExceededValueThresholdMarker with an OR of this subset to the children list
-            if (!Constants.ANY_FIELD.equals(field)
+            if (!skip
```

Comment:
Could you update the comment to describe the significance of the additional `skip` boolean?